dist/index.html

```diff
@@ -28,87 +28,85 @@
         <span class="menu-btn__burger"></span>
       </div>
       <div class="nav-container lato_regular">
-        <img class="logo" src="./img/logo/logo-black.png" alt="" />
+        <a class="menu-link" href="./index.html"><img class="logo" src="./img/logo/logo-black.png" alt="Anime Fans Logo"/></a>
         <nav class="nav">
           <ul class="menu-nav">
```

Comment:
- [ ] [Optional] Kindly add a link to the home to make it easier for users to navigate through pages.

Reply:
Yes sir I will do it, thank you very much